### PR TITLE
Name change

### PR DESCRIPTION
**Pull request by JnHs**
"Managed services" has a specific meaning as seen in https://docs.microsoft.com/en-us/azure/lighthouse/concepts/managed-services-offers etc - hoping to reword this to avoid confusion and be clearer that this refers to PaaS.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@JnHs : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by JnHs** [comment]
@ktoliver I still don't see this live at https://docs.microsoft.com/en-us/azure/architecture/guide/design-principles/managed-services, does something more need to happen? thanks for the help!

**Comment by ktoliver** [comment]
Hi @JnHs - I'm not involved in publishing...

@MicrosoftDocs/pubdesk @MicrosoftDocs/pubdesk_admins Could you investigate this report that revisions made in the public architecture-center repo 8 days ago are not reflected on the live site? Thanks.
